modules/microshift-olm-namespaces.adoc

```diff
@@ -0,0 +1,29 @@
+//Module included in the following assemblies:
+//
+//* microshift_running_apps/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-olm-namespaces_{context}"]
+= Namespace details as used in {microshift-short}
+
+The `microshift-olm` RPM creates the three default namespaces: one for running OLM, and two for catalog and Operator installation. You can create additional namespaces as relevant for your use case.
+
+.Default namespaces created by OLM for {microshift-short}
+[cols="2",%autowidth]
+|===
+|*Default Namespace*
+|*Details*
+
+|`openshift-operator-lifecycle-manager`
+|The OLM package manager runs in this namespace.
+
+|`openshift-marketplace`
+|The global namespace. Empty by default. This namespace must be set in the catalog source YAML if you want the catalog source to be available globally to users in all namespaces. An Operator group in this namespace is set to *All Namespaces* mode by default.
```

Comment:
Reword as: "If you want the catalog source to be available globally to users in all namespaces, this namespace must be set in the catalog source YAML."